sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -139,4 +160,84 @@ export class ScholasticStandingInstitutionsController extends BaseController {
       userToken.authorizations.getLocationsIds(),
     );
   }
+  /**
+   * Process a text file with application to be withdrawn.
+   * @param file text file content with all information needed to withdraw application.
```

Comment:
Do we comment UserToken?

Reply:
UserToken is not used still, so removed it